PRASFiles.jl/src/Systems/read.jl

```diff
@@ -299,25 +336,37 @@ Reads additional user defined metadata from the file containing the PRAS system.
 function read_addl_attrs(inputfile::String)
```

Comment:
It seems like this method does two unrelated things: print the contents of a Dict (super generic), and wrap the HDF5.File version of the method in a call to h5open. I would suggest splitting out the printing into a separate function (you don't necessarily want to print the attributes when opening the HDF5 file anyways, right?